Changes

```diff
@@ -80,6 +80,9 @@ Working version
 - #2308: More debugging information on [Cmm] terms
   (Mark Shinwell, review by Stephen Dolan)
 
+- #7878, #8542: Replaced TypedtreeIter with tast_iterator
+  (Isaac "Izzy" Avram)
```

Comment:
Please add reviewers here (@gasche and myself).

Reply:
Sure thing!